public/themes/custom/hdbt_subtheme/templates/form/details.html.twig

```diff
@@ -26,11 +26,11 @@
   %}
   <summary {{ summary_attributes.addClass(summary_classes) }}>
     <div class="accordion_heading_container">
-      <div>{{ title }}</div>
+      <div id="accordion-{{ accordionid }}-heading">{{ title }}</div>
       <span aria-hidden="true" class="hel-icon hel-icon--angle-down hel-icon--size-m"></span>
     </div>
   </summary>
-  <div id="accordion-1-content" role="region" class="Accordion-module_accordionContent__1umso" aria-labelledby="accordion-1-heading" style="">
+  <div id="accordion-{{ accordionid }}-content" role="region" class="Accordion-module_accordionContent__1umso" aria-labelledby="accordion-{{ id }}-heading" style="">
```

Comment:
should the `{{ id }}`  also be `{{ accordionid }}`?

Reply:
Yes, fixed that.